src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js

```diff
@@ -63,14 +63,14 @@ class SkillFeedbackCard extends Component {
     this.setState({ newFeedbackValue: event.target.value });
   };
 
-  handleEditOpen = () => {
+  handleEditOpen = previousFeedback => {
     this.handleMenuClose();
     this.props.actions.openModal({
       modalType: 'editFeedback',
       handleConfirm: this.postFeedback,
       handleClose: this.props.actions.closeModal,
       errorText: this.state.errorText,
-      feedback: this.state.newFeedbackValue,
+      feedback: previousFeedback,
```

Comment:
This approach won't work. You need to initialise the `newFeedbackValue` state variable with `data.feedback` to achieve the desired behaviour.

Reply:
@akshatnitd once user has posted a feedback, then `newFeedbackValue` is required only when user tries to edit it. So why not I intialise it when user clicks the edit button.